samples/canopen/canopen_rpdo/RPDOCanNode.cpp

```diff
@@ -0,0 +1,30 @@
+#include "RPDOCanNode.hpp"
```

Comment:
```suggestion
#include <RPDOCanNode.hpp>
```

Reply:
Changing to angled brackets makes the header file not resolvable